sample/src/main/java/com/chuckerteam/chucker/sample/HttpBinClient.kt

```diff
@@ -120,4 +139,27 @@ class HttpBinClient(
             }
         })
     }
+
+    private class HttpBinSampleCookieJar : CookieJar {
```

Comment:
I don't think that we need this thing at all with some specifically set data.
 
If you want to set some custom cookies just use that provided url from `httpbin.org` and put data into the query there.